userdocs/src/getting-started/application-addons.md

```diff
@@ -0,0 +1,36 @@
+To help manage the application lifecycle we provide addons that aim to make your life easier as a developer. While there aren't many of these available at the moment, we expect this to change as new requirements in the product teams surface.
+
+## Postgres
+
+It is now possible to create an [AWS RDS Postgres](https://aws.amazon.com/rds/postgresql/) database from the cluster declaration.  By adding the following section to your scaffolded `cluster.yaml` file:
+
+```yaml
+metadata:
+  environment: test # Just here to clarify the example
+databases:
+  postgres:
+    - name: dbtest
+      user: administrator
+      namespace: dbtest
+```
+
+Followed by applying the updated declaration with:
+
+```bash
+$ okctl apply cluster -f cluster.yaml
+```
+
+This will create a Postgres database in your AWS account together with a number of related resources:
+
+- A secrets manager secret for the administrator password
+- Lambda function that rotates the administrator password every 30 days
+- A security group that allows Pods that have that association to connect with the RDS Postgres database
```

Comment:
I have written out parts of this mostly for "us", my hope is that by using scaffolding it won't be necessary for most users to care about this at all. However, we do have "easy to look under the hood" as part of our mission statement, which is why this documentation exists. I am not sure if I am answering your critique here at all by saying this though 🤔 What do you feel is missing? A 6, or however many, step list if a user wants to do this on their own?

Reply:
I see +1 I agree, that is useful.

This specific page part of the "getting started"-guide, which in my mind is supposed to be a simple guide with steps for how to get the user up and running. For a user that doesn't know what a Config map is, it would be nice with something that tells him/her that the PGUSER can be found as environment variable (or file for the config map perhaps).

But we have a task to go over documentation for 1.0, so we can wait with this for that task.